scripts/lint.sh

```diff
@@ -29,14 +29,7 @@ if rustup component add clippy; then
   cargo_feature opentelemetry-otlp "http-proto, surf-client, surf/curl-client"
   cargo_feature opentelemetry-otlp "metrics"
 
-  cargo_feature opentelemetry-jaeger "surf_collector_client, surf/curl-client"
-  cargo_feature opentelemetry-jaeger "isahc_collector_client"
-  cargo_feature opentelemetry-jaeger "reqwest_blocking_collector_client"
-  cargo_feature opentelemetry-jaeger "reqwest_collector_client"
-  cargo_feature opentelemetry-jaeger "collector_client"
-  cargo_feature opentelemetry-jaeger "wasm_collector_client"
-  cargo_feature opentelemetry-jaeger "collector_client, wasm_collector_client"
-  cargo_feature opentelemetry-jaeger "default"
+  cargo_feature opentelemetry-jaeger "full"
```

Comment:
This is nice to have the `full` feature, but we probably want to keep those separate tests to make sure those feature can also work on their own

Reply:
Done.